### DISCUSSION TITLE
Facing issue with Mentions example.

### DISCUSSION THREAD

**Discussion by vkbansal** [ianstormtaylor/slate#5353] [Q&A]
I tried recreating a simple mentions example here https://codesandbox.io/s/slatejs-poc-9j4f20?file=/src/App.tsx

The issue that I am facing is that when ever I delete any void element, all of them are deleted.

![Kapture 2023-03-10 at 18 49 48](https://user-images.githubusercontent.com/5930351/224326416-459c9af2-7216-4411-a875-1324e58cb909.gif)

Any help would be greatly appreciated! 🙏🏽

**Comment by jannikbuschke** [comment]
For me your problem does not show up, using edge
